src/events/reactionAdded.js

```diff
@@ -48,11 +48,15 @@ export default async ({ event }) => {
         user
       );
     } else if (message) {
+
+      let update;
+
       if (!message.files || message.files.length == 0) {
```

Comment:
This is just a general rule of mine - when I see two lines in a conditional that nearly match such as what is contained here, I usually extract the difference into a variable and then condense the code to one call.  For instance, you could refactor this to:

```
let hasNoFiles = !message.files || message.files.length == 0
let files = hasNoFiles ? [] : message.files
let update = await createUpdate(files, channel, ts, user, message.text)
```

I would argue ^ is more readable & concise, also doesn't require the predefined-and-then-mutated variable 'update'